src/config.rs

```diff
@@ -79,7 +83,15 @@ pub struct Config {
     pub(crate) target: LevelFilter,
     pub(crate) target_padding: TargetPadding,
     pub(crate) location: LevelFilter,
+    #[cfg(feature = "chrono")]
+    pub(crate) time_format: Cow<'static, str>,
+    #[cfg(feature = "chrono")]
+    pub(crate) time_offset: FixedOffset,
```

Comment:
Same with the offset. Maybe we could make this a light wrapper, that has a `UTC` constant (like both `time::UtcOffset` and `chrono::FixedOffset` have) and a constructor `from_offset(hours: i8)` or something? Depending on the feature enabled, that one could also implement either `From<time::UtcOffset>` or `From<chrono::FixedOffset>`.